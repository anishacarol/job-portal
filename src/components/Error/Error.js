import React from "react";
import * as Styled from "./Error.styles";

const Error = () => (
  <Styled.Error data-testid="error">
    Something went wrong. Please try again!
  </Styled.Error>
);

export default Error;
