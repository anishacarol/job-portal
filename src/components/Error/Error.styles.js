import styled from "styled-components";

export const Error = styled.h4`
  color: red;
`;
