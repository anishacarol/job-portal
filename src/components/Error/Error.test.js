import { render } from "@testing-library/react";
import React from "react";
import Error from "./Error";

describe("<Error/>", () => {
  it("renders Error component properly", () => {
    const { asFragment } = render(<Error />);

    expect(asFragment()).toMatchSnapshot();
  });
});
