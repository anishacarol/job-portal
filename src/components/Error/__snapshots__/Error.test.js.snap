// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error/> renders Error component properly 1`] = `
<DocumentFragment>
  <h4
    class="sc-AxjAm gxygnu"
    data-testid="error"
  >
    Something went wrong. Please try again!
  </h4>
</DocumentFragment>
`;
